ansible/roles/schulcloud-server-core/templates/preview-generator-scaled-object.yml.j2

```
@@ -0,0 +1,21 @@
+---
+apiVersion: keda.sh/v1alpha1
+kind: ScaledObject
+metadata:
+  name: rabbitmq-scaledobject
+  namespace: {{ NAMESPACE }}
+spec:
```

add the label for the app to it
```suggestion
metadata:
  name: rabbitmq-scaledobject
  namespace: {{ NAMESPACE }}
  labels:
    app: preview-generator
spec:
```